Shopify/tophat
# Deprecate support for API endpoint artifact retrieval

### What does this change accomplish?

Using an API endpoint on a remote server can be handy, but it does require an organization to potentially need to lower their security standards on the web service that hosts these endpoints to just basic HTTP authentication.  This change deprecates and removes the ability to retrieve artifacts this way in favour of the classic URL method which doesn't require additional APIs to be exposed on potentially sensitive services.

### How have you achieved it?

By deleting all API endpoint based artifact retrieval.

### How can the change be tested?

1. Build and run Tophat.
2. Ensure there that the API endpoint based retrieval is gone from the app.
